### PR TITLE
release: v1.9.0

### PR DESCRIPTION
**Pull request by abernix**
# 👀 

> **Note**
>
> This PR is just a merge commit.  The PR worth reviewing is #2455, which will squash merge into this.
> This PR will true merge to `main`.

### CODE REVIEW

File: NEXT_CHANGELOG.md [hunk 1]
```diff
@@ -23,5 +21,211 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
...
+### Fix `APOLLO_USAGE_REPORTING_INGRESS_URL` behavior when Router was run without a configuration file
+
+The environment variable `APOLLO_USAGE_REPORTING_INGRESS_URL` (not usually necessary under typical operation) was **not** being applied correctly when the Router was run without a configuration file.
+In addition, defaulting of environment variables now directly injects the variable rather than injecting via expansion expression.  This means that the use of `APOLLO_ROUTER_CONFIG_ENV_PREFIX` (even less common) doesn't affect injected configuration defaults.
+
+By [@bryncooke](https://github.com/bryncooke) in https://github.com/apollographql/router/pull/2432
+
+## 🛠 Maintenance
+
+### Remove unused factory traits ([Issue #2180](https://github.com/apollographql/router/pull/2372))
```
Comment (Geal) [inline NEXT_CHANGELOG.md:181]:
```suggestion
### Remove unused factory traits ([PR #2372](https://github.com/apollographql/router/pull/2372))
```

Comment (abernix) [reply NEXT_CHANGELOG.md:181]: Thanks for reviewing this here.  I will mark this as resolved here and apply this the prep PR that I opened subsequently! (#2455)

**Comment by Geal** [comment]
o/ could you cherry-pick https://github.com/apollographql/router/commit/419a6ed5fda948cb6ef800229ebe0f2136b635d5 which fixes https://github.com/apollographql/router/pull/2447/commits/7c055f5b5c073482089f336a8b0329b873cbd5cd ?

**Comment by abernix** [comment]
> o/ could you cherry-pick [419a6ed](https://github.com/apollographql/router/commit/419a6ed5fda948cb6ef800229ebe0f2136b635d5) which fixes [7c055f5](https://github.com/apollographql/router/commit/7c055f5b5c073482089f336a8b0329b873cbd5cd) ?

done, @Geal

**Review by Geal — APPROVED** [review]

**Review by BrynCooke — APPROVED** [review]

**Pull request merged**